Practicum/Pract. 03/tasks-solutions/zad3.cpp

```diff
@@ -0,0 +1,70 @@
+#include <iostream>
+#include <fstream>
+#include <sstream>
+#define STUDENTS_MAX 1024
+#define NAME_MAX_LENGTH 16
+
+enum class HairColor {
+	BROWN,
+	BLACK,
+	BLONDE,
+	RED,
+	WHITE
+};
```

Comment:
The good practice is to set the values to the enum strings even though they have default values.